routing/router.go

```diff
@@ -874,20 +874,27 @@ func (r *ChannelRouter) pruneZombieChans() error {
 
 	// With the set of zombie-like channels obtained, we'll do another pass
 	// to delete them from the channel graph.
-	toPrune := make([]uint64, 0, len(chansToPrune))
-	for chanID := range chansToPrune {
-		toPrune = append(toPrune, chanID)
-		log.Tracef("Pruning zombie channel with ChannelID(%v)", chanID)
-	}
-	if err := r.cfg.Graph.DeleteChannelEdges(toPrune...); err != nil {
-		return fmt.Errorf("unable to delete zombie channels: %v", err)
-	}
+	if len(chansToPrune) > 0 {
```

Comment:
nit: just return if len==0 so we can save some indenting?

Reply:
Yeah, that's much better. Diff also much smaller 🤩